pyServer/AzureDiskInspectService.py

```diff
@@ -336,6 +342,14 @@ def do_POST(self):
             else:
                 runWithCredscan = False
 
+            if b'timeout' in postvars:
+                timeoutInMinsStr = str(postvars[b'timeout'][0], encoding='UTF-8')
+                timeoutInMins = int(timeoutInMinsStr)
```

Comment:
Sounds good!

Reply:
Decided to use timeoutInMinsStr.isdecimal() instead since isnumeric() would also allow fractions and super/sub-scripts.
https://www.programiz.com/python-programming/methods/string/isdecimal